deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```diff
@@ -96,26 +96,29 @@
   value: 3600
 
 {{- range $keyLabel := .Values.ccdb.encryption.rotation.key_labels }}
-{{- if not (regexMatch "^[a-z]+[a-z0-9_]*[a-z0-9]+$" $keyLabel) }}
-{{ fail "One or more items in ccdb.encryption.rotation.key_labels don't match the regex /^[a-z]+[a-z0-9_]*[a-z0-9]+$/" }}
+{{- if not (regexMatch "^[a-zA-Z]+[a-zA-Z0-9_]*[a-zA-Z0-9]+$" $keyLabel) }}
+{{ fail "One or more items in ccdb.encryption.rotation.key_labels don't match the regex /^[a-zA-Z]+[a-zA-Z0-9_]*[a-zA-Z0-9]+$/" }}
 {{- end }}
 {{- if gt (len $keyLabel) 240 }}
 {{ fail "One or more items in ccdb.encryption.rotation.key_labels exceed the maximum length of 240 characters" }}
 {{- end }}
 - type: replace
   path: /variables/-
   value:
-    name: "ccdb_key_label_{{ $keyLabel }}"
+    {{- /* Don't double-prefix labels that already have the prefix because they are
+           imported from an earlier version of kubecf.  This allows upgrades to the
+           new (verbatim) label naming scheme while retaining access to old key values. */ }}
```

Comment:
I believe this is failing `helm lint`: `unexpected comment ends before closing delimiter in input`
It expects either `{{- /*` / `*/ -}}` or `{{/*` / `*/}}` (spaces are significant here).

Reply:
So `helm lint` requires you to violate the [best practices](https://helm.sh/docs/chart_best_practices/templates/#formatting-templates) for formatting templates from its own documentation:

>Template directives should have whitespace after the opening braces and before the closing braces

So yeah, will change to `{{- /* ... */}}` even though I think this is wrong.